src/app/auth/settings/account-settings.component.ts

```diff
@@ -23,12 +25,15 @@ export class AccountSettingsComponent extends BaseComponent implements OnInit {
 
     public activeTab: AccountSettingsTabs = AccountSettingsTabs.PROFILE;
     public user$: Observable<MaybeNull<AccountFragment>>;
+    public adminPrivelegesOn: boolean = false;
```

Comment:
Don't manage state in a component. Always in a service

Reply:
Perhaps `LoggedUserService` should encapsulate this state.
Do not work with low level `LocalStorageService` here.